libs/application/templates/new-primary-school/src/fields/FriggOptionsAsyncSelectField/index.tsx

```diff
@@ -32,6 +33,13 @@ const FriggOptionsAsyncSelectField: FC<
   const { title, props, defaultValue, id, marginBottom } = field
   const { isMulti = false, optionsType, placeholder } = props
 
+  let friggOptionsType: OptionsType
+  if (typeof optionsType === 'function') {
+    friggOptionsType = optionsType(application)
+  } else {
+    friggOptionsType = optionsType
+  }
```

Comment:
Can be simplified to
```suggestion
  let friggOptionsType = optionsType
  if (typeof optionsType === 'function') {
    friggOptionsType = optionsType(application)
  }
```